client/src/components/Form/Form.js

```diff
@@ -0,0 +1,156 @@
+import React, {useState} from 'react';
+import {Button, TextField, Alert, AlertTitle, styled} from '@mui/material';
+
+import {ColorRing} from 'react-loader-spinner';
+import {useFormik} from 'formik';
+import * as yup from 'yup';
+import * as emailjs from 'emailjs-com';
+
+const AlertMessage = styled((props) => <Alert {...props}/>)(({theme}) => ({
```

Comment:
You can make it simpler, because you don't pass any props here
```suggestion
const AlertMessage = styled(Alert)(({theme}) => ({
```